### DISCUSSION TITLE
posts/618

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#2] [Announcements]
# posts/618

Golang으로 프로그램을 개발하다 보면, 어느 순간 GC(Garbage Coll...

https://tech.kakao.com/posts/618

**Comment by sejin-P** [comment]
안녕하세요, 좋은 글 재밌게 읽었습니다. 다만 4번 단락의 frame graph는 혹시 flame graph를 잘못 말씀하신걸까요?

**Reply by sue-cream** [reply]
안녕하세요! frame -> flame 수정했습니다, 감사합니다! 😊

**Comment by hellower** [comment]
쌍따봉 날립니다!
